package.json

```diff
@@ -21,7 +21,7 @@
   "homepage": "https://github.com/CheaprEats/js-sdk#readme",
   "devDependencies": {
     "babel-cli": "6.26.0",
-    "babel-core": "6.26.0",
+    "babel-core": "^6.26.3",
```

Comment:
Ran `npm audit fix`. We need to find a way to fix the github security alert we get for lodash